ironfish/src/mining/director.ts

```diff
@@ -463,20 +463,22 @@ export class MiningDirector {
       return MINED_RESULT.INVALID_BLOCK
     }
 
-    this.logger.info(
-      `Successfully mined block ${block.header.hash.toString('hex')} (${
-        block.header.sequence
-      }) has ${block.transactions.length} transactions`,
-    )
-
-    const { isAdded, reason } = await this.chain.addBlock(block)
+    const { isAdded, reason, isFork } = await this.chain.addBlock(block)
 
     if (!isAdded) {
       this.logger.error(`Failed to add mined block to chain with reason ${String(reason)}`)
       return MINED_RESULT.ADD_FAILED
     }
 
-    this.blocksMined++
+    if (!isFork) {
+      this.logger.info(
+        `Successfully mined block ${block.header.hash.toString('hex')} (${
+          block.header.sequence
+        }) has ${block.transactions.length} transactions`,
+      )
+
+      this.blocksMined++
```

Comment:
Let's inverse this so it just returns and doesn't gossip a block on a fork.

Reply:
Do you mean adding a new MINED_RESULT such as `MINED_RESULT.FORK` and return that in case of `isFork`

Reply:
something like that? 
`if (isFork) return MINED_RESULT.FORK`